abci/types/messages.go

```diff
@@ -123,6 +123,22 @@ func ToRequestFinalizeBlock(req *RequestFinalizeBlock) *Request {
 	}
 }
 
+//
```

Comment:
If we keep the comments, can we add the following?
`// TODO CHECK HEIMDALL-V2 check implementation of SideChannel`

Reply:
Same, not needed anymore.